src/components/contributions-table.jsx

```diff
@@ -114,7 +122,61 @@ class ContributionsTable extends React.Component {
       return 0;
     };
 
+    function parseElectionName(input) {
+      const [ename, ...rest] = input.split('-');
+
+      if (rest.length === 1) {
+        // If there is only one element after the split, assume it's the year
+        return { ename, month: 'november', year: rest[0] };
+      } else if (rest.length >= 2) {
+        // If there are two or more elements, assume the last is the year
+        // and the second-to-last is the month
+        return { ename, month: rest[rest.length - 2], year: rest[rest.length - 1] };
+      }
+      // Handle the case where there are no elements after the split
+      return { ename, month: 'november', year: undefined };
+    }
+
+    const electionCompare = (x, y) => {
+      const election1 = parseElectionName(x);
+      const election2 = parseElectionName(y);
+
+      if (election1.name > election2.name) {
```

Comment:
I'm confused how this is working. It looks to me like parseElectionName returns an object with a property `ename`, rather than `name`. Is it possible these cases are never hit because it's always comparing undefined to undefined?
What's the desired behavior here, that elections would sort alphabetically by locale?